storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -27,18 +27,30 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.uber.org/multierr"
 
+	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/prompb"
 	"github.com/prometheus/prometheus/util/annotations"
 )
 
+type ResourceAttributeAction int
+
+const (
+	PromoteResourceAttributeAction ResourceAttributeAction = iota
+	IgnoreResourceAttributeAction
+)
+
+type ResourceAttributesSetting struct {
+	Action ResourceAttributeAction
+	Attr   map[string]struct{}
```

Comment:
```suggestion
	Attr   map[string]struct{}{}
```

you meant to create a Set here, right? Or am I missing something?

Reply:
yes, it is a set.
But that extra `{}` is not need it at `struct` definition. As you can see in https://www.bytesizego.com/blog/set-in-golang

<img width="527" alt="image" src="https://github.com/user-attachments/assets/4044da85-dbc0-4cb2-b217-329540ca7395" />